apps/angular/src/app/features/anime/anime-dashboard/anime-dashboard.component.ts

```diff
@@ -22,10 +33,88 @@ export class AnimeDashboardComponent {
 		'status',
 	];
 
+	/** Initial anime sorting. */
+	protected readonly initialSortingParameters: SortingParameters<AnimeSortingField>;
+
+	/** Anime filters form group. */
+	protected readonly animeFiltersFormGroup: FormGroup<{
+		search: FormControl<string | null>;
+		animeTypes: FormControl<readonly string[] | null>;
```

Comment:
```suggestion
		animeTypes: FormControl<string[] | null>;
```

We don't normally describe outgoing arrays as `readonly`. In the future, this may cause some difficulties when you do want to change these arrays.